INTERPRETING.md

```diff
@@ -64,16 +64,15 @@ properties of the global scope prior to test execution.
         6. Return Completion(status).
 
   - **`global`** - a reference to the global object on which `$262` was initially defined
-  - **`uncallableAndIsHTMLDDA`** - a function that returns an object *`obj`* for
-    which [Call](https://tc39.github.io/ecma262/#sec-call)(*`obj`*, *any value*, «»)
-    throws a `TypeError`.  (The value of [IsCallable]()(*`obj`*) is unspecified:
-    a callable *`obj`* that throws a `TypeError` or an uncallable *`obj`* works
-    equally well.)  In hosts supporting the
-    [IsHTMLDDA](https://tc39.github.io/ecma262/#sec-IsHTMLDDA-internal-slot)
-    internal slot, *`obj`* must also have such a slot.  (These highly specific
-    behaviors are entirely motivated by the very few tests that use this.  Read
-    them for an explanation.)  Tests that use this function should be marked as
-    using the `uncallableAndIsHTMLDDA` feature.
+  - **`fakeDocumentAll`** - an object that 1) optionally has an [[IsHTMLDDA]]
+    internal slot, and 2) when called with no arguments or with the single
+    argument `""` returns `null`.  Use this property to test that ECMAScript
+    algorithms aren't mis-implemented to treat `document.all` as being
+    `undefined` or of type Undefined (instead of Object).  (The peculiar second
+    requirement permits testing algorithms when they also call `document.all`
+    with such arguments, so that testing for correct behavior requires knowing
+    how the call behaves.  This is rarely necessary.)  Tests using this function
+    should be tagged with the `fakeDocumentAll` feature.
```

Comment:
while we are at it, I think just calling it `IsHTMLDDA` - instead of `fakeDocumentAll` - would be fine and it makes an easier reference for the ES specs.

This shouldn't be a big deal, but if we should also avoid rediscussion on this name here.